### ISSUE TITLE
Node must be provided when reporting error if location is not provided

### ISSUE THREAD

**Issue by carlitux** [eslint/eslint#7544]
**Tell us about your environment**

* **ESLint Version:** v3.9.1
* **Node Version:** v6.9.1
* **npm Version:** 4.0.1

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**

```json
{
  "extends": ["airbnb"],
  "env": {
    "browser": true,
    "node": true,
    "mocha": true
  },
  "parser": "babel-eslint",
  "parserOptions": {
    "sourceType": "module"
  },
  "rules": {
    "max-len": ["error", 80],
    "jsx-quotes": ["error", "prefer-single"],
    "space-before-function-paren": ["error", { "anonymous": "never", "named": "always" }],
    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
  },
  "plugins": [
    "react", "import"
  ],
  "settings": {
    "ecmascript": 7,
    "import/resolver": {
      "node": {
        "paths": ["src"],
        "modulesDirectories": ["node_modules"]
      }
    }
  },
  "globals": {
    "__DEVELOPMENT__": true,
    "__CLIENT__": true,
    "__SERVER__": true,
    "__DISABLE_SSR__": true,
    "__DEVTOOLS__": true,
    "socket": true,
    "mixpanel": true,
    "Raven": true,
    "webpackIsomorphicTools": true
  }
}
```

**What did you do? Please include the actual source code causing the issue.**
```js
export GlobalNotification from './GlobalNotification';
```

**What did you expect to happen?**
Report the errors and warnings.

**What actually happened? Please include the actual, raw output from ESLint.**

```zsh
Node must be provided when reporting error if location is not provided
AssertionError: Node must be provided when reporting error if location is not provided
    at EventEmitter.module.exports.api.report (/Users/carlitux/Projects/yei.bo/node_modules/eslint/lib/eslint.js:970:20)
    at RuleContext.report (/Users/carlitux/Projects/yei.bo/node_modules/eslint/lib/rule-context.js:144:21)
    at EventEmitter.Program:exit (/Users/carlitux/Projects/yei.bo/node_modules/eslint-plugin-import/lib/rules/prefer-default-export.js:69:19)
    at emitOne (events.js:101:20)
    at EventEmitter.emit (events.js:188:7)
    at NodeEventGenerator.leaveNode (/Users/carlitux/Projects/yei.bo/node_modules/eslint/lib/util/node-event-generator.js:49:22)
    at CodePathAnalyzer.leaveNode (/Users/carlitux/Projects/yei.bo/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:628:23)
    at CommentEventGenerator.leaveNode (/Users/carlitux/Projects/yei.bo/node_modules/eslint/lib/util/comment-event-generator.js:110:23)
    at Controller.leave (/Users/carlitux/Projects/yei.bo/node_modules/eslint/lib/eslint.js:930:36)
    at Controller.__execute (/Users/carlitux/Projects/yei.bo/node_modules/estraverse/estraverse.js:397:31)
```

**Comment by not-an-aardvark** [comment]
Hi,

This looks like an issue with one of [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import)'s custom rules, and not with ESLint itself. I would recommend filing an issue on that repo instead.

**Comment by carlitux** [comment]
You are right, I found it.

Thank you!

**Comment by ilyavolodin** [comment]
Closing as the question has been answered.
